graphblas/tests/test_matrix.py

```diff
@@ -384,17 +384,17 @@ def test_ewise_add(A):
         [2, 0, 1, 2, 2, 2, 3, 3, 4, 4, 5, 5, 6],
         [4, 3, 5, 3, 8, 5, 3, 7, 8, 3, 1, 7, 4],
     )
-    with pytest.raises(TypeError, match="require_monoid"):
-        A.ewise_add(B, binary.second)
+    # with pytest.raises(TypeError, match="require_monoid"):
+    A.ewise_add(B, binary.second)  # okay now
     # surprising that SECOND(x, empty) == x, which is why user
     # must opt-in to using binary ops in ewise_add
-    C = A.ewise_add(B, binary.second, require_monoid=False).new()
+    C = A.ewise_add(B, binary.second).new()
```

Comment:
Update comments. The opt-in requirement no longer exists.

Reply:
Done.  Thanks for the attention to detail!